packages/chart/src/components/LineChart/index.tsx

```diff
@@ -432,6 +433,47 @@ const LineChart = (props: LineChartProps) => {
           </Text>
         )}
       </Group>
+      <Group left={Number(config.runtime.yAxis.size)} className='glyph-tooltip-group'>
+        <Group key={`tooltip-group`} display={'block'}>
+          {/* tooltips */}
+          <Bar
+            key={'tooltip bars'}
+            width={Number(xMax)}
+            height={Number(yMax)}
+            fill={DEBUG ? 'red' : 'transparent'}
+            fillOpacity={0.05}
+            onMouseMove={showSingleSeries ? null : e => handleTooltipMouseOver(e, tableData)}
```

Comment:
I feel like this is a bit more straight forward way to write this: 

onMouseMove={ (e) => {
    if(showSingleSeries) return
    handleTooltipMouseOver(e, tableData)
  }}